doc/Caliptra.md

```diff
@@ -957,44 +1006,9 @@ Firmware signing key rotation shall follow the requirements described in [Refere
-  * 128 KiB for DCCM and 48 KiB for ROM
-* Cryptography requirements:
-  * SHA256, SHA384, and SHA512
-  * ECDSA Secp384r1 with HMAC-DRBG - key generation, signing and verification
-  * HMAC SHA384
-  * AES256-CBC
-* Chips Alliance is used for RISC-V.
-* APB is the choice for the SoC-facing interface.
-* JTAG is exported at the IP interface.
-* TRNG is the digital logic and algorithms that are required for random number generation. It requires a physical entropy source input. See the Caliptra IP specification and [integration specification](https://github.com/chipsalliance/caliptra-rtl/blob/main/docs/CaliptraIntegrationSpecification.md) for more information.
-  * For SoCs that want to use their own legacy TRNG, Caliptra provides a HW API to push the random number on the request/response handshake. See the Caliptra IP specification and [integration specification](https://github.com/chipsalliance/caliptra-rtl/blob/main/docs/CaliptraIntegrationSpecification.md) for more information. **This mode is advised for early development but discouraged for production tape outs due to the lower security assurances of an external TRNG**.
-
-The following figure describes the Caliptra IP HW boot flow.
-
-*Figure 9: Hardware boot flow*
-
-![](./images/Caliptra_boot_flow2.png)
+Please refer to Caliptra HW specification -> https://github.com/chipsalliance/caliptra-rtl/blob/main/docs/CaliptraHardwareSpecification.md 
 
-1. As part of the SoC boot flow, SoC may have other infrastructure and entities that boot. That part of the flow is outside the scope of this document. If SoC chooses to bypass Caliptra, then it should have a capability to bypass Caliptra entirely through its proprietary flow. This may be needed for initial power-on and other early validation.
-2. Cptra\_pwrgood is asserted to the Caliptra IP block.
-3. Cptra\_rst\_b is deasserted to the Caliptra IP block. See the integration specification for guidelines on the minimum number of cycles between these two signals.
-4. Caliptra IP now evaluates the strap settings driven through various interface wires (for example, the security or debug state of the SoC).
-5. If SoC is in a debug mode, then security assets are cleared or switched to debug mode.
-6. Caliptra IP asserts Ready\_For\_Fuse wire to the SoC.
-7. The SoC populates the fuse registers, the internal TRNG configuration registers, and the ROM WDT cycle count, then sets the CPTRA\_FUSE\_WR\_DONE bit. Note that Caliptra HW drops writes to any registers that cannot be changed unless there is a power cycle (for example, UDS). So SoC is free to write all the registers.
-8. Caliptra IP deasserts Ready\_for\_Fuse wire as soon as the fuse write done register is written.
-9. Caliptra IP moves security critical assets in fuse registers (for example, UDS) to the key vault.
-
-## Caliptra FW push flow
+## Caliptra FW push flow for Passive/Legacy Mode
```

Comment:
nit: Just use the term "Passive" mode, rather than "Legacy". We want to make it clear that both modes will remain supported.

Reply:
Fixed legacy name; but as I commented above to Varun, passive mode cannot run UDS manuf flows because it doesnt have the necessary HW interfaces namely AXI-MGR. This was the decision we took few months ago. AXI-SUB cannot initiate a transaction and that is just how ARM HW spec is defined.

Reply:
Also cant respond to the hash calculation HW API comment, but same comment as above and I wrote more explanation and reasons why we did that.

Reply:
Oh, let's discuss the UDS manufacturing flows in the other comment thread. Closing this one.